### PR TITLE
Convert absolute imports to relative ones

### PR DESCRIPTION
**Pull request by smashercosmo**
**Summary**

Convert the rest of the absolute imports to relative ones and remove babel-plugin-module-resolver as it's not needed anymore. Part of https://github.com/netlify/netlify-cms/pull/5374

**Test plan**

Run current tests and try to build all the packages

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -25,55 +25,6 @@ const defaultPlugins = [
...
-            Formats: path.join(__dirname, 'packages/netlify-cms-core/src/formats/'),
-            Integrations: path.join(__dirname, 'packages/netlify-cms-core/src/integrations/'),
-            Lib: path.join(__dirname, 'packages/netlify-cms-core/src/lib/'),
-            Reducers: path.join(__dirname, 'packages/netlify-cms-core/src/reducers/'),
-            Selectors: path.join(__dirname, 'packages/netlify-cms-core/src/selectors/'),
-            ReduxStore: path.join(__dirname, 'packages/netlify-cms-core/src/redux/'),
-            Routing: path.join(__dirname, 'packages/netlify-cms-core/src/routing/'),
-            ValueObjects: path.join(__dirname, 'packages/netlify-cms-core/src/valueObjects/'),
-            localforage: 'localforage',
-            redux: 'redux',
```
Comment (smashercosmo) [inline babel.config.js:72]: Have no idea, why localforage and redux were aliased. But seems like it works fine without it.

File: packages/netlify-cms-core/src/actions/collections.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import history from '../routing/history';
+import { history } from '../routing/history';
```
Comment (smashercosmo) [inline packages/netlify-cms-core/src/actions/collections.ts:1]: Converted history to named export

File: packages/netlify-cms-core/src/bootstrap.js [hunk 3]
```diff
@@ -2,18 +2,18 @@ import React from 'react';
...
-import { getPhrases } from 'Lib/phrases';
-import { selectLocale } from 'Reducers/config';
-import { I18n } from 'react-polyglot';
 import { GlobalStyles } from 'netlify-cms-ui-default';
-import { ErrorBoundary } from 'UI';
-import App from 'App/App';
-import 'EditorWidgets';
-import 'coreSrc/mediaLibrary';
+import { I18n } from 'react-polyglot';
+import { store } from './redux';
```
Comment (smashercosmo) [inline packages/netlify-cms-core/src/bootstrap.js:7]: Converted redux store to named export

**Review by erezrokah — APPROVED** [review]
Thanks @smashercosmo for another great PR.

I went ahead and enabled `eslint-plugin-import` to catch import bugs.

**Review by erezrokah — APPROVED** [review]

**Pull request merged**